webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -433,9 +433,12 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
     (cPage: number, cPageSize: number) => {
       updateSettings({ pageLimit: cPageSize });
       // Pagination component is assuming starting index of 1.
+      if (cPage - 1 !== page) {
```

Comment:
super nit: right now there's a split second where the rows get unselected before the page changes and it looks a little odd. maybe try switching the order of this if statement and the setPage?

Reply:
Pretty sure that won't help. The selection is updated as an effect of the `clearSelectionTrigger` changing [here](https://github.com/determined-ai/determined/blob/ea57f640240591f627a997df98547eadd08ff7b6/webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx#L222-L225). ideally, the selection should be lifted up into the `F_ExperimentList` component and derived from the `selectedExperimentIds` array, but that's out of scope for this pr.

Reply:
I'm not sure I understand, that still seems like it would work to me. I would expect that if you called setPage first then the page would change before clearSelectionTrigger triggers that useEffect. Regardless, it's not a big deal.

Reply:
Ah, ok, that makes sense. It looks like that new commit does fix the issue!